### PR TITLE
Add min gas option 

### LINKED ISSUE #1467: Document `tx-pool-min-gas-price` option
Document new `--tx-pool-min-gas-price` option, which is decoupled from the existing `--min-gas-price` option. See https://github.com/hyperledger/besu/pull/6098 for more information.

### PR DESCRIPTION
**Pull request by joaniefromtheblock**
Close #1467

### CODE REVIEW

File: docs/public-networks/reference/cli/options.md [hunk 1]
```diff
@@ -4983,6 +4983,47 @@ instead of the number of transactions.
...
 :::
 
+### `tx-pool-min-gas-price`
+
+<Tabs>
+
+<TabItem value="Syntax" label="Syntax" default>
+
+```bash
+--tx-pool-min-size=<INTEGER>
```
Comment (macfarla) [inline docs/public-networks/reference/cli/options.md:4993]:
```suggestion
--tx-pool-min-gas-price=<INTEGER>
```

File: docs/public-networks/reference/cli/options.md [hunk 2]
```diff
@@ -4983,6 +4983,47 @@ instead of the number of transactions.
...
+```
+
+</TabItem>
+
+<TabItem value="Example" label="Example">
+
+```bash
+--tx-pool-min-size=2000
```
Comment (macfarla) [inline docs/public-networks/reference/cli/options.md:5001]:
```suggestion
--tx-pool-min-gas-price=2000
```

File: docs/public-networks/reference/cli/options.md [hunk 3]
```diff
@@ -4983,6 +4983,47 @@ instead of the number of transactions.
...
+```bash
+BESU_TX_POOL_MIN_SIZE=2000
+```
+
+</TabItem>
+
+<TabItem value="Configuration file" label="Configuration file">
+
+```bash
+tx-pool-max-size="2000"
```
Comment (macfarla) [inline docs/public-networks/reference/cli/options.md:5017]:
```suggestion
tx-pool-min-gas-price="2000"
```

File: docs/public-networks/reference/cli/options.md [hunk 4]
```diff
@@ -4983,6 +4983,47 @@ instead of the number of transactions.
...
+```bash
+--tx-pool-min-size=2000
+```
+
+</TabItem>
+
+<TabItem value="Environment variable" label="Environment variable">
+
+```bash
+BESU_TX_POOL_MIN_SIZE=2000
```
Comment (macfarla) [inline docs/public-networks/reference/cli/options.md:5009]:
```suggestion
BESU_TX_POOL_MIN_GAS_PRICE=2000
```

**Review by macfarla — COMMENTED** [review]
default is 1000 so 2000 seems to be a sensible example suggestion

File: docs/public-networks/reference/cli/options.md [hunk 5]
```diff
@@ -4983,6 +4983,47 @@ instead of the number of transactions.
...
+
+```bash
+tx-pool-max-size="2000"
+```
+
+</TabItem>
+
+</Tabs>
+
+The minimum gas price required for a transaction to be accepted into the [transaction pool](../../concepts/transactions/pool.md).
```
Comment (fab-10) [inline docs/public-networks/reference/cli/options.md:5024]: Suggestion to add the unit

```suggestion
The minimum gas price, in wei, required for a transaction to be accepted into the [transaction pool](../../concepts/transactions/pool.md).
```

**Review by fab-10 — APPROVED** [review]

**Review by mjsmike62 — APPROVED** [review]
LGTM

**Review by alexandratran — APPROVED** [review]

**Pull request merged**